src/linear_algebra/finite_dimensional.lean

```diff
@@ -896,6 +896,20 @@ calc  findim K V
 ... = findim K f.range : by rw [ker_eq_bot.2 hf, findim_bot, add_zero]
 ... ≤ findim K V₂ : submodule.findim_le _
 
+/-- Given a linear map `f` between two vector spaces with the same dimension, if
+`ker f = ⊥` then `linear_equiv_of_ker_eq_bot` is the induced isomorphism
+between the two vector spaces. -/
+noncomputable def linear_equiv_of_ker_eq_bot
+  [finite_dimensional K V] [finite_dimensional K V₂]
+  (f : V →ₗ[K] V₂) (hf : f.ker = ⊥) (hdim : findim K V = findim K V₂) : V ≃ₗ[K] V₂ :=
+linear_equiv.of_bijective f hf (linear_map.range_eq_top.2 $
+  (linear_map.injective_iff_surjective_of_findim_eq_findim hdim).1 (linear_map.ker_eq_bot.1 hf))
```

Comment:
Thank you for adding this!  I should have done this before asking you, but let's just confirm with @Vierkantor that it doesn't already exist in some form.

Reply:
The results I'm aware of are `injective_iff_surjective_of_findim_eq_findim` and `linear_equiv_of_is_basis`, so this looks like a good addition to me :+1: